lib/kafka-consumer.js

```diff
@@ -1,46 +1,44 @@
 /**
  * @fileOverview Wrapper for node-rdkafka Consumer Ctor, a mixin.
  */
-
-var Promise = require('bluebird');
-var cip = require('cip');
-var kafka = require('node-rdkafka');
-
-var magicByte = require('./magic-byte');
-var log = require('./log.lib').getChild(__filename);
+const Promise = require('bluebird');
+const cip = require('cip');
+const kafka = require('node-rdkafka');
+const magicByte = require('./magic-byte');
```

Comment:
Hey Ples, very nice job!! I just want to chime in to mention the logic behind how the requires are done, they are split into three blocks:

```
[Node.js dependencies (e.g. require('util'))]

[Third Party dependencies (e.g. require('axios'))]

[Local dependencies (e.g. require('./local'))]
```

Each block is split by a newline.

Cheers

Reply:
Thank you, changed.